### ISSUE TITLE
publish new otel-lgtm image with latest grafana ( 11.1.0)

### ISSUE THREAD

**Issue by madhub** [grafana/docker-otel-lgtm#76]
publish new otel-lgtm image with latest grafana ( 11.1.0)

I tried building local docker image , it build successfully  but fails to run with error **"/bin/sh: /otel-lgtm/run-all.sh: /bin/bash^M: bad interpreter: No such file or directory"**

```shell
cd docker
docker build . -t grafana/otel-lgtm
docker run --name lgtm -p 3000:3000 -p 4317:4317 -p 4318:4318 --rm -ti grafana/otel-lgtm:latest
```
### Error during docker run
![image](https://github.com/user-attachments/assets/82a62422-9397-4dde-ba71-eedc6371ca0d)

Tested on 
WIndows 11 With Docker Desktop version **4.31.1**

docker info
```text
Server:
 Containers: 52
  Running: 51
  Paused: 0
  Stopped: 1
 Images: 63
 Server Version: 26.1.4
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: d2d58213f83a351ca8f528a95fbd145f5654e957
 runc version: v1.1.12-0-g51d5e94
 init version: de40ad0
 Security Options:
  seccomp
   Profile: unconfined
 Kernel Version: 5.15.153.1-microsoft-standard-WSL2
 Operating System: Docker Desktop
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 7.609GiB
 Name: docker-desktop
 ID: 5c78005b-8f21-4360-b074-0ce826847485
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: http.docker.internal:3128
 HTTPS Proxy: http.docker.internal:3128
 No Proxy: hubproxy.docker.internal
 Labels:
  com.docker.desktop.address=npipe://\\.\pipe\docker_cli
 Experimental: false
 Insecure Registries:
  hubproxy.docker.internal:5555
  127.0.0.0/8
 Live Restore Enabled: false
```

**Comment by zeitlinger** [comment]
@fstab fyi

**Comment by madhub** [comment]
@zeitlinger  I can build & run myself if the bug reported below is fixed 


_I tried building local docker image , it build successfully but fails to run with error "/bin/sh: /otel-lgtm/run-all.sh: /bin/bash^M: bad interpreter: No such file or directory"_
```

cd docker
docker build . -t grafana/otel-lgtm
docker run --name lgtm -p 3000:3000 -p 4317:4317 -p 4318:4318 --rm -ti grafana/otel-lgtm:latest
```

Error during docker run
![350667932-82a62422-9397-4dde-ba71-eedc6371ca0d](https://github.com/user-attachments/assets/d26f6dba-0027-4ad1-88a5-7e236de88603)

**Comment by zeitlinger** [comment]
I think this will be fixed with https://github.com/grafana/docker-otel-lgtm/pull/79

**Comment by madhub** [comment]
> I think this will be fixed with #79

Yes, it works. I can my local build. Thanks
